aptos-move/aptos-vm-types/src/storage.rs

```diff
@@ -253,10 +253,10 @@ impl StoragePricing {
                     gas_params,
                 )),
             },
-            10.. => V2(StoragePricingV2::new_without_storage_curves(
```

Comment:
remove `new_without_storage_curves`?

Reply:
https://github.com/aptos-labs/aptos-core/blob/137acee4c6dddb1c86398dce25b041d78a3028d3/aptos-move/aptos-vm-types/src/storage.rs#L429 change this to V3 probably?

Reply:
Yeah, trying this right now

Reply:
Done.